oryx-app-common/src/test/java/com/cloudera/oryx/app/kmeans/KMeansPMMLUtilsTest.java

```diff
@@ -62,54 +47,11 @@ public void testClustering() {
 
   @Test
   public void testPMMLToClusters() {
-    PMML pmml = buildDummyClusteringModel();
+    PMML pmml = KMeansPMMLUtils.buildDummyClusteringModel();
     List<ClusterInfo> clusterInfoList = KMeansPMMLUtils.read(pmml);
     assertEquals(NUM_CLUSTERS, clusterInfoList.size());
     assertEquals(NUM_DIMENSIONS, clusterInfoList.get(0).getCenter().length);
     assertEquals(CLUSTER_SIZE, clusterInfoList.get(1).getCount());
   }
 
-  public static PMML buildDummyClusteringModel() {
```

Comment:
They r both pretty much identical, so was looking to drop one of them.  The method in KMeansPMMLUtils is also called in 2 other tests in Speed and Batch layers,  hence it kind'a seemed to make sense to drop this and retain the one in non-test source.